gh-actions-scripts/build_helm_chart.sh

```diff
@@ -42,7 +42,7 @@ fi
 mv "${IMAGE}-${VERSION}.tgz" "installer/${IMAGE}-${VERSION}.tgz"
 
 #verify the chart
-helm template "installer/${IMAGE}-${VERSION}.tgz" --dry-run > /dev/null
+helm template "installer/${IMAGE}-${VERSION}.tgz" --dry-run --set remoteControlPlane.api.token=thisisnotavalidkeptntoken > /dev/null
```

Comment:
If we want to verify the chart why didn't we use a [helm lint](https://helm.sh/docs/helm/helm_lint/) command before packaging ?  I know it didn't change in this PR, just curious :thinking:

Reply:
Didn't know that linting existed, I will adapt the current script to use it ... the edit was a pretty ugly fix to get things working

Reply:
The initial thought here (coming from Keptn core, waaaay back...) was to verify that the helm chart can be rendered. Not sure if `helm lint` also does the job.

Reply:
`helm lint` checks that the chart is formally correct, `helm template` shows you what the content of the rendered chart would be but we are throwing away those to `/dev/null`
I am not saying that we need to change, I was just wondering why one was used instead of the other...